src/components/01-atoms/06-call-to-action/call-to-action.njk

```diff
@@ -1,6 +1,4 @@
 {% if not standAlone %}<span></span>{% endif %}
-<p class="wp-block-button{%if class %} {{ class }}{% endif %}">
-	<a href="{{ href }}" class="wp-block-button__link">{{ label }}
-		{% if hasIcon %}{% render '@svg', {svg: 'external'}, true %}{% endif %}
-	</a>
+<p>
+	<a {% if external %}rel="external" {% endif %}href="{{ href }}" class="cta{% if modifier %} cta--{{ modifier }}{% endif %}">{{ label }}{% if external %}{% render '@svg', {svg: 'external'}, true %}{% endif %}</a>
```

Comment:
The external link icon is not being rendered.

Reply:
Hey sorry @jhung, forgot to link to that component. The default is internal, the external one is https://deploy-preview-218--pinecone.netlify.com/components/preview/call-to-action--external